hack/build/build.sh

```diff
@@ -9,15 +9,19 @@ fi
 
 TAG="$1"  #TODO: validate tag is vX.Y.Z
 COMPONENT="kubevirt-template-validator"
-BRANCH=$( git rev-parse --abbrev-ref HEAD )
-REVISION=$( git rev-parse --short HEAD )
+LDFLAGS="\
+-X 'github.com/fromanirh/kubevirt-template-validator/internal/pkg/version.COMPONENT=$COMPONENT' \
```

Comment:
please use `github.com/kubevirt` not `github.com/fromanirh`

Reply:
If I do that the variables do not get set.
IIUC in order to do have that change, there must be more changes that are out of the scope of this PR.